src/main/resources/com/sonymobile/jenkins/plugins/lenientshutdown/ShutdownConfiguration/config.jelly

```diff
@@ -1,4 +1,4 @@
-/*
+<!--
```

Comment:
Probably because I was more familiar with the jelly stuff. Changed it back to groovy